### ISSUE TITLE
generate.STL

### ISSUE THREAD

**Issue by robjhyndman** [tidyverts/fabletools#291]
`generate.STL()` could be a nice way to include the functionality of `forecast::bld.mbb.bootstrap()`. It would be different from the other `generate.xxx` methods in fable that the time index would be the same as the training data, rather than future to the training data.

As a side issue, it seems we have lost the ability to generate from other models with `future = FALSE`.

**Comment by robjhyndman** [comment]
I can have a go at writing `generate.STL()` if you agree in principle to this idea.

**Comment by mitchelloharawild** [comment]
I agree with this in principle as a user-customisable `generate()` method.
As far as I can tell, it should be very easy to add and I can do tomorrow if you like.

`future = FALSE` should be possible by specifying the time values you'd like to generate, via `generate(<mable>, new_data = <tsibble>)`. In theory, this should allow you to request sampled values/paths for specific time points, however individual model support for this detail is lacking (more history of the fit is required, which would also be useful for speeding up `hfitted()` by avoiding refitting).

**Comment by robjhyndman** [comment]
Sounds good. Yes, I think the default new_data should be the same period as the training data for the STL model.
